### PR TITLE
Task/77 localization

### PR DESCRIPTION
**Pull request by jvrastil**
- all strings should be localized now (except some placeholders which are temporary)
- coin names and tickers are displayed in latin letters
- language can be switched in settings panel
- user settings should be stored into database so it is restored after login again

### CODE REVIEW

File: lib/logic/account_cubit/account_cubit.dart [hunk 1]
```diff
@@ -24,7 +26,16 @@ class AccountCubit extends Cubit<AccountState> {
   Account? loadAccount(String accountId) {
     final Account? account = _accountRepository.get(accountId);
 
-    if (account == null) return null;
+    if (state.account?.id == accountId) return state.account;
+
+    if (account == null) {
```
Comment (storkandstars) [inline lib/logic/account_cubit/account_cubit.dart:31]: Please make a separate logic for clearing out an account from memory.

File: lib/main.dart [hunk 2]
```diff
@@ -67,51 +69,65 @@ class _SimplioAppState extends State<SimplioApp> {
...
-                  BlocProvider(
-                    create: (context) => AccountCubit.builder(
-                      accountRepository:
-                          RepositoryProvider.of<AccountRepository>(context),
-                      assetWalletRepository:
-                          RepositoryProvider.of<AssetWalletRepository>(context),
-                    )..loadAccount(state.accountId),
+        ],
+        child: MultiBlocListener(
+          listeners: [AuthBlocListeners.clearAccountAfterLogout],
```
Comment (storkandstars) [inline lib/main.dart:82]: Logic of a this listener is already in AuthGuard. Put clear account logic to `onUnauthenticated` callback

File: lib/main.dart [hunk 3]
```diff
@@ -67,51 +69,65 @@ class _SimplioAppState extends State<SimplioApp> {
...
+
+                      return AuthenticatedScreen(
+                        navigatorKey: AuthenticatedRoute.key,
+                        initialRoute: AuthenticatedRoute.home,
+                        onGenerateRoute: _authenticatedRouter.generateRoute,
+                      );
+                    },
+                  );
+                },
+                onUnauthenticated: (context) => Navigator(
```
Comment (storkandstars) [inline lib/main.dart:125]: clear account here

File: lib/logic/account_cubit/account_cubit.dart (hunk 1 above)
Comment (jvrastil) [reply lib/logic/account_cubit/account_cubit.dart]: fixed

File: lib/main.dart (hunk 2 above)
Comment (jvrastil) [reply lib/main.dart]: fixed

**Review by storkandstars — APPROVED** [review]

**Review by storkandstars — APPROVED** [review]

**Pull request merged**